boards/arm/nucleo_f207zg/nucleo_f207zg.dts

```diff
@@ -130,6 +130,20 @@
 		     &eth_txd1_pb13>;
 };
 
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/* Set 2KB of storage at the end of 1MB flash */
```

Comment:
The last sector of STM32F2xx is 128KB not 2KB

Reply:
Good catch, thanks @ABOSTM